packages/react-cookie/src/UniversalCookies.ts

```diff
@@ -0,0 +1,54 @@
+import cookie from 'cookie';
+import {Context} from 'koa'
+import {Cookie, CookieValue} from './types';
+
+export {CookieContext} from './context';
+
+export class UniversalCookies {
```

Comment:
open to better names here

Reply:
This one only works on the server right? ServerCookieManager?